packages/core/src/codewhisperer/service/transformByQ/transformApiHandler.ts

```diff
@@ -488,6 +636,7 @@ export async function pollTransformationJob(jobId: string, validStates: string[]
                 result: MetadataResult.Pass,
             })
             status = response.transformationJob.status!
+            console.log('status = ' + status)
```

Comment:
Is temporary for debugging? If not use getLogger()

Reply:
Yes just temporary